docs/excel/custom-functions-authentication.md

```diff
@@ -55,9 +55,9 @@ function StoreValue(key, value) {
 }
 ```
 
-When the task pane needs the access token, it can retrieve the token from `AsyncStorage`. The following code sample shows how to use the `AsyncStorage.getItem` method to retrieve the token.
+When the task pane needs the access token, it can retrieve the token from `Office.Storage`. The following code sample shows how to use the `Office.Storage.getItem` method to retrieve the token.
 
-```javascript
+```js
```

Comment:
and this one should use Office.Storage. I would check all the code samples to be sure they got updated.